src/main/java/io/managed/services/test/client/securitymgmt/SecurityMgmtApi.java

```
@@ -1,60 +1,51 @@
-            var ex = (ApiException) e;
-            return new ApiUnknownException(ex.getMessage(), ex.getCode(), ex.getResponseHeaders(), ex.getResponseBody(), ex);
-        }
-        return null;
-    }
-
-    @Override
-    protected void setAccessToken(String accessToken) {
-        ((HttpBearerAuth) apiClient.getAuthentication("Bearer")).setBearerToken(accessToken);
+        this.v1 = apiClient.api().kafkas_mgmt().v1();
     }
 
     @SuppressWarnings("unused")
     public ServiceAccount getServiceAccountById(String id) throws ApiGenericException {
-        return retry(() -> api.getServiceAccountById(id));
+        return retry(() -> v1.service_accounts(id).get().get(10, TimeUnit.SECONDS));
     }
 
-    public ServiceAccountList getServiceAccounts() throws ApiGenericException {
-        return retry(() -> api.getServiceAccounts(null));
+    public ServiceAccountListResponse getServiceAccounts() throws ApiGenericException {
+        return retry(() -> v1.service_accounts().get().get(10, TimeUnit.SECONDS));
     }
 
     public ServiceAccount createServiceAccount(ServiceAccountRequest serviceAccountRequest) throws ApiGenericException {
-        return retry(() -> api.createServiceAccount(serviceAccountRequest));
+        return retry(() -> v1.service_accounts().post(serviceAccountRequest).get(10, TimeUnit.SECONDS));
     }
 
     public void deleteServiceAccountById(String id) throws ApiGenericException {
         // TODO: why does it return Error
-        retry(() -> api.deleteServiceAccountById(id));
+        retry(() -> v1.service_accounts(id).delete().get(10, TimeUnit.SECONDS));
     }
 
     public ServiceAccount resetServiceAccountCreds(String id) throws ApiGenericException {
-        return retry(() -> api.resetServiceAccountCreds(id));
+        return retry(() -> v1.service_accounts(id).reset_credentials().post().get(10, TimeUnit.SECONDS));
+    }
+
+    @Override
+    protected ApiUnknownException toApiException(Exception e) {
+        return null;
```

this doesn't sound right, here we are swallowing exceptions.